Shared/Samples/Show device location using indoor positioning/ShowDeviceLocationUsingIndoorPositioningView.swift

```diff
@@ -0,0 +1,103 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import CoreLocation
+import SwiftUI
+
+struct ShowDeviceLocationUsingIndoorPositioningView: View {
+    /// The data model for the sample.
+    @StateObject private var model = Model()
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    var body: some View {
+        MapView(map: model.map)
+            .locationDisplay(model.locationDisplay)
+            .overlay(alignment: .top) {
+                HStack(alignment: .center, spacing: 10) {
+                    VStack(alignment: .leading) {
+                        // The floor number of a location when in a building
+                        // where the ground floor is 0. Negative numbers
+                        // indicate floors below ground level.
+                        if let floor = model.currentFloor {
+                            let humanReadableFloor = if floor >= .zero {
+                                // Add 1 to above the ground floor levels.
+                                floor + 1
+                            } else {
+                                // Don't change the underground floor levels.
+                                floor
+                            }
+                            Text("Current floor: \(humanReadableFloor)")
+                        }
+                        if let accuracy = model.horizontalAccuracy {
+                            let formatStyle = Measurement<UnitLength>.FormatStyle.measurement(
+                                width: .abbreviated,
+                                usage: .asProvided,
+                                numberFormatStyle: .number.precision(.fractionLength(2))
+                            )
+                            Text("Accuracy: \(Measurement<UnitLength>(value: accuracy, unit: .meters), format: formatStyle)")
+                        }
+                    }
+                    .opacity(model.currentFloor != nil ? 1 : 0)
+                    VStack(alignment: .leading) {
+                        let sourceType = model.positionSource == "GNSS" ? "Satellites" : "Transmitters"
+                        Text("Data source: \(model.positionSource ?? "None")")
+                        Text("Number of \(sourceType): \(model.signalSourceCount ?? 0)")
+                    }
+                }
+                .frame(maxWidth: .infinity)
+                .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
+            }
+            .overlay(alignment: .center) {
+                if model.isLoading {
+                    ProgressView(
+                        """
+                        Loading
+                        indoor data…
+                        """
+                    )
+                    .padding()
+                    .background(.ultraThinMaterial)
+                    .cornerRadius(10)
```

Comment:
`cornerRadius(_:antialiased:)` is deprecated in iOS 18, but I am okay if you want to leave as is for now.

Reply:
Will fix.
Can you log an issue in this repo so we can clean them up later? Thanks.